extensions/hadoop/src/main/java/com/hazelcast/jet/hadoop/HadoopSources.java

```diff
@@ -171,7 +171,7 @@ private HadoopSources() {
             @Nonnull ConsumerEx<Configuration> configureFn,
             @Nonnull BiFunctionEx<K, V, E> projectionFn
     ) {
-        return Sources.batchFromProcessor("readHadoop", readHadoopP(configureFn, projectionFn));
+        return Sources.batchFromProcessor("readHadoop", readHadoopP(null, configureFn, projectionFn));
```

Comment:
I think read file permission should be used for this too

Reply:
My idea was to create specific permissions for non-OOTB connectors like Hadoop, Kafka, Elastic.. in the next release.